kickstart.ks

```diff
@@ -0,0 +1,28 @@
+ostreesetup --osname=rhel --url=file:///run/install/repo/ostree/repo --ref=rhel/9/x86_64/edge --nogpg --remote=edge
+reboot
+zerombr
+clearpart --all --initlabel
+part /boot/efi --fstype=efi --size=200
+part /boot --fstype=xfs --asprimary --size=800
+part swap --fstype=swap --recommended
+part pv.01 --grow
+volgroup rhel pv.01
+network --bootproto=dhcp
+logvol / --vgname=rhel --fstype=xfs --size=10000 --name=root
+%post --log=/var/log/anaconda/post-install.log --erroronfail
+
+useradd -m -d /home/redhat -p '$6$DYgjv/BVpdq/0EVt$2fd9RPHleTgsFWzTLL/I.znl9vbKgt00eXQ0LNbkc7wBF67fSYBsZd6LutDZHI0YZNg3SKB04SdpLOkuWRzni.' -G wheel redhat
+echo -e 'redhat\tALL=(ALL)\tNOPASSWD: ALL' >> /etc/sudoers
+
+# Make sure redhat user directory contents ownership is correct
+chown -R redhat:redhat /home/redhat/
+
+# Add the pull secret to CRI-O and set root user-only read/write permissions
+cat << EOF > /etc/crio/openshift-pull-secret
```

Comment:
How does this work if the pull secret is empty and never written?

Reply:
ack this makes sense to use jinja for